task.js

```diff
@@ -1 +1,20 @@
-const add = require('./commands/add.js')
+const fs = require('fs')
+const add = require('./commands/add')
+const list = require('./commands/list')
+const done = require('./commands/done')
+
+const command = process.argv[2]
+const string = process.argv[3]
```

Comment:
What is `string`? Generic variable names are hard to reason about, because they don't give much information. :(

To make it more obvious what the second argument is, you could do something like this:

```js
const command = process.argv[2]
const argument = process.argv[3]

switch(command) {
  case "add":
    const task = argument
    add(task)
    break;

  case "list":
    list()
    break;

  case "done":
    const taskID = argument
    done(taskID)
}
```